src/video/windows/SDL_windowswindow.c

```diff
@@ -518,6 +539,14 @@ WIN_CreateWindow(_THIS, SDL_Window * window)
         return WIN_SetError("Couldn't create window");
     }
 
+    if (SDL_GetHintBoolean(SDL_HINT_WINDOWS_DWM_DARK_MODE, SDL_FALSE)) {
+        BOOL value = TRUE;
+        if (!pDwmSetWindowAttribute)
+            LoadDWMFunctions();
```

Comment:
It's probably not worth creating a separate function in SDL_windowswindow.c to load just one function. Go ahead and do the loadobject/loadfunction inline here, as we do a bunch of other places.